graphrag/index/flows/create_final_communities.py

```diff
@@ -61,6 +63,12 @@ def create_final_communities(
 
     filtered["title"] = "Community " + filtered["id"].astype(str)
 
+    # Add period timestamp to the community reports
+    filtered["period"] = datetime.now(timezone.utc).date().isoformat()
+
+    # Add size of the community
+    filtered["size"] = filtered.loc[:, "text_unit_ids"].apply(lambda x: len(x))
```

Comment:
I think you can change it to `.apply(len)` no need for the lambda

Reply:
I'll submit another PR optimizing all this calculations. This change was to attack some warnings being output